src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -92,11 +93,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
         String user = userService.getCurrentUser().getEmail();
 
-        String text = Jsoup.clean(request.getParameter("text"), Whitelist.none());
```

Comment:
I just added this line this afternoon, why didn't you have HTML tag when you remove this?

Reply:
The original function of keeping html tags in the message is to preserve the style (such as <h1></h1> <p></p>) so that when we display the stored message, the style would be the same. Adding Whitelist.none() removes that function since it filters out all html tags. The display problem is actually because of how we render the message element when we display it on public feed. Originally, the starter code used document.createTextNode(), where you can only set the nodeValue, which is text only. I changed it to document.createElement() in feed.html so that you can set the stored message with html tags as innerHTML instead. That way the html tags would serve their purpose instead of being treated as regular text.